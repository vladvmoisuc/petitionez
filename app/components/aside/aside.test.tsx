import { render } from "@testing-library/react";

import Aside from ".";

describe("<Aside />", () => {
  it("should match the snapshot", () => {
    const { container } = render(
      <Aside>
        <span>Test...</span>
      </Aside>
    );

    expect(container).toMatchSnapshot();
  });
});
