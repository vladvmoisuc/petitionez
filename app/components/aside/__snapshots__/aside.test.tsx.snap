// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Aside /> should match the snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1hu82mc-MuiContainer-root"
  >
    <div
      class="MuiBox-root css-8kjoqe"
    >
      <span>
        Test...
      </span>
    </div>
  </div>
</div>
`;
